README.md

```diff
@@ -1,40 +1,41 @@
 # Welcome to the lab on enabling GitHub Pages!
 
-- [About the codebase](#about-the-codebase)
-- [Lab instructions](#lab-instructions)
-- [Grading criteria](#grading-criteria)
-- [A note about usage of GitHub Actions](#a-note-about-usage-of-github-actions)
-- [References](#references)
+This lab is an exercise to check your knowledge on enabling GitHub Pages. It is automatically graded via a workflow once you have completed the instructions.
 
+**Quick links:**
+- [About this lab](#about-this-lab)
+- [Instructions](#instructions)
+- [Seeing your result](#seeing-your-result)
+- [Troubleshooting](#troubleshooting)
+- [Useful resources](#useful-resources)
```

Comment:
Not necessarily for this PR, but I'd love for us to find a way to automate this TOC. On my machine, I use the TOC extension for VSCode, but I also know an action exists that'll add the TOC for us. Although the Readme and TOC will likely be the same for every repo so maybe it's a moot point.